### PR TITLE
Update Translations Portuguese-BR 

### PR DESCRIPTION
**Pull request by crashzk**

### CODE REVIEW

**Comment by crashzk** [comment]
**@R1KO** I believe that it can be merged without problems.

Added Portuguese-BR translation, tested it on my servers, all right.

File: addons/sourcemod/data/vip/cfg/groups.ini [hunk 1]
```diff
@@ -1,14 +1,12 @@
 "VIP_GROUPS"
 {
-	"vip1" // Имя группы
+	"vip1" 	// Group name
```
Comment (R1KO) [inline addons/sourcemod/data/vip/cfg/groups.ini:3]: `// Group name / Имя группы`

let's make all comments in 2 languages: russian and english

**Review by R1KO — CHANGES_REQUESTED** [review]

File: addons/sourcemod/data/vip/cfg/groups.ini (hunk 1 above)
Comment (crashzk) [reply addons/sourcemod/data/vip/cfg/groups.ini:3]: Okay, I will be making these changes and I will return here so that I can do the review.

**Comment by crashzk** [comment]
@R1KO I believe that everything is right now, anything just let me know.

File: addons/sourcemod/data/vip/cfg/info.ini [hunk 2]
```diff
@@ -2,24 +2,26 @@
 {
 	"connect_info_time"	// Сообщение для VIP-игрока при входе на сервер (когда вип на время)
 	{
-		//	{NAME}	- Ник игрока
-		//	{GROUP}	- Группа игрока
+		//	{NAME}		- Ник игрока
```
Comment (R1KO) [inline addons/sourcemod/data/vip/cfg/info.ini:5]: comments in this file also need to be done in two languages

**Review by R1KO — CHANGES_REQUESTED** [review]

File: addons/sourcemod/data/vip/cfg/info.ini (hunk 2 above)
Comment (crashzk) [reply addons/sourcemod/data/vip/cfg/info.ini]: Done

**Review by R1KO — APPROVED** [review]

**Pull request merged**